dns/unbound-plus/src/opnsense/scripts/OPNsense/Unboundplus/dnsbl.sh

```diff
@@ -213,16 +213,19 @@ simpletrack() {
 
 install() {
         # Put all files in correct format
+        WHITE=$(cat ${DESTDIR}/whitelist.inc | tr ',' '|')
+        
         for FILE in $(find ${WORKDIR} -type f); do
-		WHITE=$(cat ${DESTDIR}/whitelist.inc | tr ',' '|')
 		if [ -z "${WHITE}" ]; then
-			cat ${FILE} | sort -u | awk '{printf "server:\n", $1; printf "local-data: \"%s A 0.0.0.0\"\n", $1}' > ${FILE}.inc
+			awk '{printf "server:\n", $1}' > ${FILE}.inc
```

Comment:
This line does not work. Regarding how list is still accepted, seems this line can be eliminated?